src/commands/deploy/showDeployCompletedMessage.ts

```diff
@@ -9,16 +9,21 @@ import { AppServiceDialogResponses } from "../../constants";
 import { SiteTreeItem } from "../../explorer/SiteTreeItem";
 import { ext } from "../../extensionVariables";
 import { localize } from "../../localize";
+import { uploadAppSettings } from "../appSettings/uploadAppSettings";
 import { startStreamingLogs } from '../logstream/startStreamingLogs';
 
 export function showDeployCompletedMessage(node: SiteTreeItem): void {
-    const message: string = localize('deployCompleted', 'Deployment to "{0}" completed.', node.client.fullName);
+    const message: string = localize('deployCompleted', 'Deployment to "{0}" completed. ([View Output](command:{1}))', node.client.fullName, ext.prefix + '.showOutputChannel');
```

Comment:
How come this was added? I don't see it on the design comment above. It's basically a fourth button, and if we're not okay with a fourth button, I don't think we should be okay with a link here either.

Reply:
As discussed offline, removing "View Output" completely from the post-deploy message. 
- "View Output" in this post-deploy scenario had low usage data and "View Output" is already available during deployment and with the deployment failed message.